modular_skyrat/modules/goofsec/code/sol_fed.dm

```diff
@@ -215,12 +548,186 @@ GLOBAL_LIST_INIT(emergency_responders, list())
+	type_to_check = /datum/antagonist/ert/request_911
+	type_of_callers = "911_responders"
+	announcement_source = "Sol Federation S.W.A.T."
+	fine_station = TRUE
+	ghost_poll_msg = "The Sol-Fed 911 services have requested a S.W.A.T. backup. Do you wish to become a S.W.A.T. member?"
+	amount_to_summon = 6
+	type_to_summon = /datum/antagonist/ert/request_911/condom_destroyer
+	summoned_type = "swat"
+	announcement_message = "Hello, crewmembers. Our emergency services have requested S.W.A.T. backup, either for assistance doing their job due to crew \
+		impediment, or due to a fraudulent 911 call. We have billed the station $20,000 for this, to cover the expenses of flying a second emergency response to \
+		your station. Please comply with all requests by said S.W.A.T. members."
+
+/obj/item/solfed_reporter/swat_caller/questions(mob/user)
+	var/question = "Does the situation require additional S.W.A.T. backup, involve the station impeding you from doing your job, \
+		or involve the station making a fraudulent 911 call and needing an arrest made on the caller?"
+	if(tgui_input_list(user, question, "S.W.A.T. Backup Caller", list("Yes", "No")) != "Yes")
+		to_chat(user, "You decide not to request S.W.A.T. backup.")
+		return FALSE
+	message_admins("[ADMIN_LOOKUPFLW(user)] has voted to summon S.W.A.T backup.")
+	return TRUE
+
+/obj/item/solfed_reporter/treason_reporter
+	name = "Treason Reporter"
+	desc = "Use this in-hand to vote that the station is engaging in Treason. If your entire team votes for it, the Military will handle the situation."
+	type_to_check = /datum/antagonist/ert/request_911/condom_destroyer
+	type_of_callers = "swat"
+	announcement_source = "Sol Federation National Guard"
+	fine_station = FALSE
+	ghost_poll_msg = "The station has decided to engage in treason. Do you wish to join the Sol Federation Military?"
+	amount_to_summon = 12
+	type_to_summon = /datum/antagonist/ert/request_911/treason_destroyer
+	summoned_type = "national_guard"
+	announcement_message = "Crewmembers of the station.\n\
+		You have refused to comply with first responders and SWAT officers, and have assaulted them, and they are unable to \
+		carry out the wills of the Sol Federation, despite residing within Sol Federation borders.\n\
+		As such, we are charging those responsible with Treason. The penalty of which is death, or no less than twenty-five years in Superjail.\n\
+		Treason is a serious crime. Our military forces are en route to your station. They will be assuming direct control of the station, and \
+		will be evacuating civilians from the scene.\n\
+		Comply, or be shot.\n\
+		Non-offending citizens, prepare for evacuation. Comply with all orders given to you by Sol Federation military personnel.\n\
+		To all those who are engaging in treason, lay down your weapons and surrender. Refusal to comply may be met with lethal force."
```

Comment:
That's kind of a very long announcement, I feel like. 7 lines might be a bit of a stretch.